integration-cli/docker_cli_stack_test.go

```diff
@@ -59,13 +59,13 @@ func (s *DockerSwarmSuite) TestStackDeployComposeFile(c *check.C) {
 
 	out, err = d.Cmd("stack", "ls")
 	c.Assert(err, checker.IsNil)
-	c.Assert(out, check.Equals, "NAME        SERVICES\n"+"testdeploy  2\n")
+	c.Assert(out, check.Equals, "NAME                SERVICES\n"+"testdeploy          2\n")
```

Comment:
Why do we need this change?
Also, it would be better to replace whitespace sequence to a single whitespace before comparison